app/controllers/users/sessions_controller.rb

```diff
@@ -1,2 +1,13 @@
 class Users::SessionsController < Devise::SessionsController
+  def new
+    super do |resource|
+      # When warden and devise threw an authentication error (invalid password)
+      # They don't respect the locale param
+      # This allows us to detect, enforce internationalization, and move the error message
```

Comment:
Would love a comment we're looking for their message to detect the authentication error